ci-operator/config/quay/quay-tests/quay-quay-tests-master__ocp-415-quay39.yaml

```diff
@@ -0,0 +1,64 @@
+base_images:
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  upi-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: quay-e2e-tests-quay39-ocp415-lp-interop
```

Comment:
Following our discussion and resolution on `quay39-ocp414` (as fixed in #45164)
I think you should change this file name to
ci-operator/config/quay/quay-tests/quay-quay-tests-master__ocp-415-quay39`-lp-interop`.yaml

For consistency :)

Reply:
NM, I got your general approach of leaving the `-lp-interop` outside of file names, since this job will be active we can use it like that. The renaming of 4.14 is just for tracking.